content/en/blog/_posts/2020-03-30-two-phased-canary-rollout-with-gloo.md

```diff
@@ -0,0 +1,772 @@
+
+```bash
+curl -sL https://run.solo.io/gloo/install | sh
+export PATH=$HOME/.gloo/bin:$PATH
+```
+
+Now, you should be able to run `glooctl version` to see that it is installed correctly:
+
+```bash
+➜ glooctl version
+Client: {"version":"1.3.15"}
+Server: version undefined, could not find any version of gloo running
+```
+
+Now we can install the gateway to our cluster with a simple command:
+
+```bash
+glooctl install gateway
+```
+
+The console should indicate the install finishes successfully:
+
+```bash
+Creating namespace gloo-system... Done.
+Starting Gloo installation...
+
+Gloo was successfully installed!
+
+```
+
+Before long, we can see all the Gloo pods running in the `gloo-system` namespace:
+
+```bash
+➜ kubectl get pod -n gloo-system
+NAME                             READY   STATUS    RESTARTS   AGE
+discovery-58f8856bd7-4fftg       1/1     Running   0          13s
+gateway-66f86bc8b4-n5crc         1/1     Running   0          13s
+gateway-proxy-5ff99b8679-tbp65   1/1     Running   0          13s
+gloo-66b8dc8868-z5c6r            1/1     Running   0          13s
+```
+
+### Deploying the application
+
+Our `echo` application is a simple container (thanks to our friends at HashiCorp) that will 
+respond with the application version, to help demonstrate our canary workflows as we start testing and 
+shifting traffic to a `v2` version of the application. 
+
+Kubernetes gives us a lot of flexibility in terms of modeling this application. We'll adopt the following 
+conventions:
+* We'll include the version in the deployment name so we can run two versions of the application 
+side-by-side and manage their lifecycle differently. 
+* We'll label pods with an app label (`app: echo`) and a version label (`version: v1`) to help with our canary rollout. 
+* We'll deploy a single Kubernetes `Service` for the application to set up networking. Instead of updating 
+this or using multiple services to manage routing to different versions, we'll manage the rollout with Gloo configuration. 
+
+The following is our `v1` echo application: 
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echo-v1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: echo
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: echo
+        version: v1
+    spec:
+      containers:
+        # Shout out to our friends at Hashi for this useful test server
+        - image: hashicorp/http-echo
+          args:
+            - "-text=version:v1"
+            - -listen=:8080
+          imagePullPolicy: Always
+          name: echo-v1
+          ports:
+            - containerPort: 8080
+```
+
+And here is the `echo` Kubernetes `Service` object:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: echo
+spec:
+  ports:
+    - port: 80
+      targetPort: 8080
+      protocol: TCP
+  selector:
+    app: echo
+```
+
+For convenience, we've published this yaml in a repo so we can deploy it with the following command:
```

Comment:
I suggest:
```suggestion
For convenience, we've published this manifest so you can deploy it with the following command:
```

and also that you put it in the website repo. See https://kubernetes.io/docs/contribute/style/write-new-topic/#including-code-from-another-file

(you can use the website repo to store the manifest even if you decide not to use a `codenew` shortcode).